tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1487,8 +1535,10 @@ def runTest(self):
         time.sleep(30)
 
         # get the current snapshot of counter values at recv and transmit ports
-        recv_counters, queue_counters = sai_thrift_read_port_counters(self.client, self.asic_type, port_list[self.src_port_id])
-        xmit_counters, queue_counters = sai_thrift_read_port_counters(self.client, self.asic_type, port_list[self.dst_port_id])
+        recv_counters, queue_counters = sai_thrift_read_port_counters(
+            self.src_client, self.asic_type, port_list['src'][self.src_port_id])
+        xmit_counters, queue_counters = sai_thrift_read_port_counters(
+            self.dst_client, self.asic-type, port_list['dst'][self.dst_port_id])
```

Comment:
typo issue: "self.asic-type",  should correct to "self.asic_type"

caused test failure as below:
```
"======================================================================",
"ERROR: sai_qos_tests.PtfReleaseBuffer",
"----------------------------------------------------------------------",
"Traceback (most recent call last):",
"  File \"saitests/py3/sai_qos_tests.py\", line 1542, in runTest",
"    self.dst_client, self.asic-type, port_list['dst'][self.dst_port_id])",
"AttributeError: 'PtfReleaseBuffer' object has no attribute 'asic'",
"",
```